Recruit-Training/Embedded-Recruit-Training/Core/Src/main.c

```diff
@@ -152,7 +164,11 @@ int main(void)
 
     /* USER CODE BEGIN RTOS_THREADS */
     //TODO: Create threads and thread attributes
+    const osThreadAttr_t blueTask_attributes = { .name = "blueTask", .priority = (osPriority_t) osPriorityNormal, .stack_size = 128 };
+    const osThreadAttr_t greenTask_attributes = { .name = "greenTask", .priority = (osPriority_t) osPriorityNormal, .stack_size = 128 };
```

Comment:
Please be consistent with formatting and follow the defaultTask_attributes' format.